bin/maintenance/update_header.py

```diff
@@ -125,6 +125,7 @@ def _update_header(project, file_path, year, substring, regex, data):
             file_write.write(content)
     elif not found:
         print(cformat('%{yellow}Missing header in %{yellow!}{}').format(os.path.relpath(file_path)))
+        return True
```

Comment:
I'd prefer an explicit `return False` in the other cases. True vs None is a bit ugly ;)

Reply:
The "Updating header" case should also cause a CI failure IMO

Reply:
Alright, I can do that. I previously made it True to avoid changing all returns (quite a few).

Reply:
I see only one in this function (plus the implicit one at the end where nothing is returned).. ;)

Reply:
I meant `update_header`. We should return True (success) if e.g. `if ext not in SUPPORTED_FILES` right?

Reply:
I'd return False ("nothing missing / to be done") in those cases, i.e. returning `True` only when there's something to be done...

Reply:
I see what you mean now.